### ISSUE TITLE
Pink text on unique Relay domain prompt does not meet AA color contrast

### ISSUE THREAD

**Issue by LZoog** [mozilla/fx-private-relay#5844]
On https://relay.firefox.com/accounts/profile/, the `#f770ff` text "With a unique ⁨Relay⁩ email domain, you can:" has a contrast ratio of 2.29:1 against the `#F9F9FB` background, and AA requires at least 4.5:1 contrast.

<img width="1037" height="431" alt="Image" src="https://github.com/user-attachments/assets/2be48280-fb2d-4719-a11e-692523989880" />
